R/conos.R

```diff
@@ -48,7 +48,7 @@ scaledMatricesP2 <- function(p2.objs, data.type, od.genes, var.scale) {
 #' @keywords internal
 scaledMatricesSeurat <- function(so.objs, data.type, od.genes, var.scale) {
   if (var.scale) {
-    warning("Seurat doesn't support variance scaling")
+    so.objs <- lapply(so.objs, function(so){ ScaleData(so, features = rownames(so))})
```

Comment:
@VPetukhov I guess we could use `sccore::plapply()` here as well. 

I'm just following the other statements, which use `lapply()`